### DISCUSSION TITLE
flux helmrelease doesn't recreate resources which are manually deleted

### DISCUSSION THREAD

**Discussion by harshadhulipala5** [fluxcd/flux2#980] [Q&A] [answered]
Hi ,

I've come up with this issue after i've accidently deleted a service and deployment which are created by helmrelease
i thought flux reconciliation will will create both the service and deployment but unfortunately that didn't create both of them 
after deleting the helmrelease ,hr got successfully reconciled again and created the resources .

can you tell me this is the expected behaviour or am i missing something 

- harsha

**Comment by hiddeco** [comment, marked as answer]
See https://github.com/fluxcd/helm-controller/issues/186#issuecomment-765667634
